doozer/doozerlib/cli/images_streams.py

```diff
@@ -807,6 +807,8 @@ def search_issues(query):
             'labels': ['art:reconciliation', f'art:package:{image_meta.get_component_name()}'],
             'versions': [{'name': release_version}],  # Affects Version/s
             'customfield_12319940': [{'name': Model(runtime.gitdata.load_data(key='bug').data).target_release[-1]}],  # customfield_12319940 is Target Version in jira
+            'customfield_12320850': ['Release Note Not Required'],  # customfield_12320850 is Release Notes Type in JIRA
```

Comment:
`customfield_12320850` is a dict and not a list. How you can see this is by using the python JIRA API and dumping out the values of a field thats already set.

```
'customfield_12320850': {
    'self': 'https://issues.redhat.com/rest/api/2/customFieldOption/31862',
    'value': 'Release Note Not Required',
    'id': '31862',
    'disabled': False
  }
```
We do not need to pass in all the values, it should probably work with this
```suggestion
            'customfield_12320850': {'value': 'Release Note Not Required'},  # customfield_12320850 is Release Notes Type in JIRA
```